### PR TITLE
#53 - Added Category entity

### PR DESCRIPTION
**Pull request by Konoplastiy**

### CODE REVIEW

File: src/main/java/com/gachigang/ontherun/common/enums/CategoryName.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.gachigang.ontherun.common.enums;
+
+public enum CategoryName {
+    SALATES,
+    BURGERS,
+    SOUPS,
+    MEATS,
+    DRINKS;
+
```
Comment (AlexandrVikhristyk) [inline src/main/java/com/gachigang/ontherun/common/enums/CategoryName.java:9]: Why do we need this method if we have name() in Emun and it is public, I believe we can simply use it

Comment (Konoplastiy) [reply src/main/java/com/gachigang/ontherun/common/enums/CategoryName.java:9]: I try to find better solution

Comment (AlexandrVikhristyk) [reply src/main/java/com/gachigang/ontherun/common/enums/CategoryName.java:9]: Just use method name() from enum instead of your custom one

Comment (Konoplastiy) [reply src/main/java/com/gachigang/ontherun/common/enums/CategoryName.java:9]: I see

**Review by AlexandrVikhristyk — APPROVED** [review]

**Pull request merged**